src/routes/choice-exercises/__tests__/ArticleChoiceExerciseScreen.spec.tsx

```diff
@@ -4,11 +4,14 @@ import React from 'react'
 
 import labels from '../../../constants/labels.json'
 import { RoutesParamsType } from '../../../navigation/NavigationTypes'
+import * as helpers from '../../../services/helpers'
 import createNavigationMock from '../../../testing/createNavigationPropMock'
 import { mockUseLoadAsyncWithData } from '../../../testing/mockUseLoadFromEndpoint'
 import wrapWithTheme from '../../../testing/wrapWithTheme'
 import ArticleChoiceExerciseScreen from '../ArticleChoiceExerciseScreen'
 
+jest.spyOn(helpers, 'shuffleArray').mockImplementation(array => {})
```

Comment:
Any specific reason why we use `spyOn` instead of `mock`? Couldn't we just do

```suggestion
jest.mock('../../../services/helpers', () => ({ shuffleArray: () => ({}) }))
```
Same for other usages.

Reply:
I am unsure about that. @SteffiStuffel @klinzo Should I change it to use `jest.mock` as proposed?

Reply:
yes please change to `.mock()`